network-api/networkapi/templates/pages/buyersguide/article_page.html

```diff
@@ -0,0 +1,57 @@
+{% extends "pages/buyersguide/base.html" %}
+
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block category_nav %}{% endblock category_nav %}
+
+{% block hero %}
+  {% if page.hero_image %}
+    {% image page.hero_image max-200x200 %}
+  {% endif %}
+{% endblock hero %}
+
+{% block guts %}
+  <div>
+    <h2>Categories</h2>
+    <ul>
+      {% for content_category_relation in page.content_category_relations.all %}
+        <li>{{ content_category_relation.content_category.title|title }}</li>
+      {% endfor %}
+    </ul>
+  </div>
+  <h1>{{ page.title }}</h1>
+  <div>
+    <h2>Authors</h2>
+    <ul>
+      {% for author_profile_relation in page.author_profile_relations.all %}
+        <li>{{ author_profile_relation.author_profile.name }}</li>
+      {% endfor %}
+    </ul>
+  </div>
+  <div>
+    <h2>Related articles (primary)</h2>
+    <ul>
+      {% for related_article in page.get_primary_related_articles  %}
+        <li>
+          <a href="{% pageurl related_article %}">{{ related_article.title }}</a>
+        </li>
+      {% endfor %}
+    </ul>
+  </div>
+  <div>
+    <h2>Body</h2>
+    {% for block in page.body %}
+        {% include_block block %}
+    {% endfor %}
+  </div>
+  <div>
+    <h2>Related articles (secondary)</h2>
+    <ul>
+      {% for related_article in page.get_secondary_related_articles  %}
+        <li>
+          <a href="{% pageurl related_article %}">{{ related_article.title }}</a>
+        </li>
+      {% endfor %}
+    </ul>
+  </div>
```

Comment:
** Suggestion **

Same as earlier comment, if some of these values, e.g `get_secondary_related_articles` are empty, we might want to hide the `h2` titles and empty `ul` elements

Reply:
Fixed in ffc06e09d3e8a8d4e81f11a476eff297efb3c55c